### DISCUSSION TITLE
Modify the login to control panel

### DISCUSSION THREAD

**Discussion by SGLara** [statamic/cms#5577] [Q&A]
Hi there, I have a quick question, is it possible to change the behavior of the default login form. What I mean is that I would like to login with a username field I have created instead of email. 

I will appreciate it anyone has an idea, thanks in advance.

<img width="475" alt="image" src="https://user-images.githubusercontent.com/58200255/159186055-d8d3cc2a-10d7-4ddd-b245-f2ff22772380.png">

**Comment by edalzell** [comment]
I don't think that's possible, you should make a [Feature Request](https://github.com/statamic/ideas/issues/new/choose)

**Reply by SGLara** [reply]
Yeah, that's something I was thinking about. Thanks @edalzell
